source/style-guide/style/text-formatting.txt

```diff
@@ -4,603 +4,1636 @@
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               The ``--ip-addresses`` option specifies the IP address
+               and an alias for the target.
+
+          - The ``--ip-addresses`` option specifies the IP address and
+            an alias for the target.
+
+   Option button names
+     Format using ``:guilabel:`` role.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               Select :guilabel:`Microsoft Exchange` and then click
+               :guilabel:`Next`.
+
+          - Select :guilabel:`Microsoft Exchange` and then click
+            :guilabel:`Next`.
+
+   Package names
+     Format using ``monospace`` text.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               You must install the ``libvirt`` package.
+
+          - You must install the ``libvirt`` package.
+
+   Page names
+     Format using ``:guilabel:`` role.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               On the :guilabel:`Preferences` page, you determine how
+               frequently you receive email about all the activity on
+               your account: daily, weekly, or both.
+
+               On the :guilabel:`Server Settings` page, click
+               :guilabel:`Check Name`, type your password, and then
+               click :guilabel:`OK`.
+
+
+          - On the :guilabel:`Preferences` page, you determine how
+            frequently you receive email about all the activity on your
+            account: daily, weekly, or both.
+
+            On the :guilabel:`Server Settings` page, click
+            :guilabel:`Check Name`, type your password, and then click
+            :guilabel:`OK`.
+
+   Panes, named and unnamed
+     Format using ``:guilabel:`` role.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               To verify that your TLS binding works, select your
+               website in the :guilabel:`Connections` pane (if it's not
+               already selected) and then click :guilabel:`Browse
+               ipAddress (https)` in the :guilabel:`Actions` pane.
+
+               In the navigation pane, select
+               :guilabel:`Composing Email`.
+
+          - To verify that your TLS binding works, select your website
+            in the :guilabel:`Connections` pane (if it's not already
+            selected) and then click :guilabel:`Browse ipAddress
+            (https)` in the :guilabel:`Actions` pane.
+
+            In the navigation pane, select :guilabel:`Composing Email`.
+
+   Parameter names
```

Comment:
If I'm guessing what you're referring to here by the rst snippets, I would call these option or argument names. Parameter names would be something you pass to a function